src/resource/resource-opening-hours/opening-period/PeriodOpeningHours.test.tsx

```diff
@@ -95,6 +95,93 @@ const testDatePeriod: DatePeriod = {
   ],
 };
 
+const testDatePeriodWithMissingStartAndEndTimes: DatePeriod = {
```

Comment:
It would generate less code if this would just be a modified version of the testDatePeriod:
```js
const testDatePeriodWithMissingStartAndEndTimes: DatePeriod = {
     ...testDatePeriod,
     timeSpanGroups: [
         { ...timeSpanGroups[0], timeSpans: [{...}, {...}] }
     ]
}
```

Reply:
- [ ] Reduce the amount of test code by applying the previously created testDatePeriod